AElf.Sdk.CSharp.Tests/TestSdkCSharpAElfModule.cs

```diff
@@ -1,26 +1,44 @@
-﻿using AElf.Database;
+﻿using System.Threading.Tasks;
+using AElf.Common;
+using AElf.Database;
+using AElf.Kernel;
 using AElf.Kernel.Infrastructure;
-using AElf.Kernel.SmartContract;
+using AElf.Kernel.SmartContract.Application;
 using AElf.Kernel.SmartContract.Contexts;
 using AElf.Modularity;
 using AElf.TestBase;
 using Microsoft.Extensions.DependencyInjection;
+using Moq;
 using Volo.Abp.Modularity;
 
 namespace AElf.Sdk.CSharp.Tests
 {
     [DependsOn(
-        typeof(DatabaseAElfModule),
         typeof(TestBaseAElfModule),
-        typeof(SmartContractAElfModule))]
-    public class TestSdkCSharpAElfModule: AElfModule
+        typeof(TestBaseKernelAElfModule))]
+    public class TestSdkCSharpAElfModule : AElfModule
     {
         public override void ConfigureServices(ServiceConfigurationContext context)
         {
             var services = context.Services;
-            context.Services.AddKeyValueDbContext<BlockchainKeyValueDbContext>(o => o.UseInMemoryDatabase());
-            context.Services.AddKeyValueDbContext<StateKeyValueDbContext>(o => o.UseInMemoryDatabase());
+            
             services.AddTransient<IStateProviderFactory, StateProviderFactory>();
+
+            services.AddTransient<ISmartContractService>(p =>
```

Comment:
do not mock

Reply:
If I use the real SmartContractService, the test project must reference the AElf.Runtime.Csharp project. But I want to test project(AElf.Sdk.CSharp) and his dependent project (AElf.Kernel.SmartContract) never reference the AElf.Runtime.CSharp project.

Reply:
@loning If do not mock ISmartContractService, I need to mock ISmartContractRunner in here.Can I modify like this?

Reply:
Mock ISmartContractService Has been removed .use mock ISmartContractRunner.